Makefile

```
@@ -146,3 +146,7 @@ test: dist/main.js $(BUILD)/test.out
 .PHONY: watch
 watch: $(BOWER_COMPONENTS) $(NODE_MODULES)
 	npx watch-exec --command 'make dist/main.js' --watch $(EXAMPLE) --watch $(SRC)
+
+.PHONY: test-watch
+test-watch: $(BOWER_COMPONENTS) $(NODE_MODULES)
+	npx watch-exec --command 'make test' --watch $(TESTS) --watch $(SRC)
```

```suggestion
	npx watch-exec --command 'make test' --watch $(TEST) --watch $(SRC)
```
I think this should be `$(TEST)` instead of `$(TESTS)`–to parallel `$(SRC)`. I think if we state the individual files with `$(TESTS)`, it passes the whole `$(TESTS)` string as an argument, but the flag will only pick up the first file in the `$(TESTS)` string. It logs out something like:
```
[watch-exec] watching: test/Test/Lynx/Form.purs, src
```
It looks like it doesn't watch any of the other files in the `test` directory.

If we use `$(TEST)` there's only one argument to pass to it and it will watch all files in the `$(TEST)` directory. It logs out something like:
```
[watch-exec] watching: test, src
```
Which watches all files in `test`.